Environment/Environment.tex

```diff
@@ -516,6 +516,13 @@ \subsection{Browsing programmatically}
 To learn more about what is available, explore the class \ct{SystemNavigation} with the browser.
 Further navigation examples can be found in the FAQ (\appref{faq}).
 
+Beside \ct{SystemNavigation}, there are also several subclasses of \clsind{Categorizer} that manage the categories in which classes or methods are organized.
+Concretely, a \clsind{ClassOrganizer} records the organization of methods in a class, and analogously, a \clsind{SystemOrganizer} records the organization of classes within system categories.
```

Comment:
Concretely, -> In particular,

Reply:
Resolved via 54bf6db8700fd02c4d21d8f3af08e68843378390